getPopsure/dirty-swan
# fix(currency-input): Fix CurrencyInput not changing when empty value

### What this PR does
This PR fixes the issue where `CurrencyInput` `onChange` was not triggering when the value was empty.
This PR also updates `CurrencyInput` to use storybook stories instead of mdx documentation.

### How to test?
Remove the first if of the condition and observe that the empty value isn't triggering:
```
 if (shadowValue === '' && value) {
      onChange?.(null)
    } else if (shadowValue) {
      onChange?.(reverseFormatInput(shadowValue));
    }
```

### Checklist:

  Or give a reason why this does not apply:
  Or give a reason why this does not apply:

### Browser support

My code works in the following browsers:
